pyproject.toml

```diff
@@ -25,9 +25,15 @@ packaging = "^20.4"
 [tool.poetry.dev-dependencies]
 black = "^19.10b0"
 mypy = "*"
+isort = "^5.6"
 
 [tool.poetry.scripts]
 brainframe = "brainframe.cli.main:main"
 
 [tool.black]
 line-length = 79
+
+[tool.isort]
```

Comment:
Glad you found that isort can be configured in the `pyproject.toml` now :+1:

Reply:
Yeah! I was very happy to see that.